styles/rule.css

```diff
@@ -119,3 +124,30 @@ a {
   color: var(--vscode-testing-iconFailed);
   padding-right:5px;
 }
+
+
+.code-difference-scrollable .code-removed {
+  background-color: var(--vscode-diffEditor-removedLineBackground);
+  padding-left: 1.5rem;
+  margin-left: -1.5rem;
+  margin-right: -1.5rem;
+  border-radius: 0px;
```

Comment:
Maybe these could be refactored to a separate class since they are shared with `.code-added`.